src/components/ContactsList.vue

```diff
@@ -154,7 +155,12 @@ export default {
 <style lang="scss" scoped>
 // Make virtual scroller scrollable
 .contacts-list {
-	max-height: calc(100vh - var(--header-height));
+	max-height: calc(100vh - var(--header-height) - 48px);
 	overflow: auto;
 }
+
+// Add empty header to contacts-list that solves overlapping of contacts with app-navigation-toogle
```

Comment:
TODO: move the new button in here instead of appNavigatin.
Feel free to do this straight away like in https://github.com/nextcloud/contacts/blob/cc1abdad941efb989cc3a83ea554522bb7961419/src/components/MemberList.vue#L40-L44

Reply:
Hm... I am pretty sure that I am not able to do that because it would require big refactoring because of moving the button out of the appnavigation-sidebar, no? 

So I'd vote for doing that in a follow-up. Are you fine with that? :)

Reply:
Depends on @ChristophWurst

Reply:
:shrug: